spark-etl/src/main/scala/geotrellis/spark/etl/s3/SpaceTimeGeoTiffS3Input.scala

```diff
@@ -7,9 +7,9 @@ import geotrellis.spark._
 import org.apache.spark.SparkContext
 import org.apache.spark.rdd.RDD
 
-class SpaceTimeGeoTiffS3Input extends S3Input[SpaceTimeInputKey, SpaceTimeKey] {
+class SpaceTimeGeoTiffS3Input extends S3Input[TemporalProjectedExtent, SpaceTimeKey] {
```

Comment:
SpaceTimeGeoTiffS3Input -> TemporalGeoTiffS3Input (and similar renames, look for SpaceTimeGeoTiff*)